packages/edge-bundler/node/formats/tarball.ts

```diff
@@ -87,71 +88,47 @@ export const bundle = async ({
       '--quiet',
       '--code-splitting',
       '--outdir',
-      distDirectory,
+      bundleDir.path,
```

Comment:
This was wrong. `distDirectory` is for the final artefacts, not for any intermediate files that are needed to construct it, which is the case of the bundled JS.